share/revng/test/configuration/revng-qa/architectures.yml

```diff
@@ -30,47 +30,48 @@ tags:
       QEMU_NAME: arm
       CFLAGS:
         - -Wl,-Ttext-segment=0x20000
+      COMMON_FLAGS:
         - -DTARGET_arm
   - name: x86-64
     implies: [cross-compiler]
     variables:
       TRIPLE: x86_64-gentoo-linux-musl-
       QEMU_NAME: x86_64
       OBJDUMP_FLAGS: -Mintel
-      CFLAGS:
+      COMMON_FLAGS:
```

Comment:
Yep, exactly. there's not much point of duplicating all the `-DSOMETHING` flags since they work the same way on both toolchains.

👌 on `GCC_CFLAGS` and `COMMON_CFLAGS`.

Reply:
Done.